litmus/skelUtil.ml

```diff
@@ -47,7 +47,7 @@ let dump_global_type loc t = match t with
 
 let rec nitems t = match t with
 | Array (_,sz) -> sz
-| Volatile t|Atomic t -> nitems t
+| Volatile t|Atomic t|Const t -> nitems t
 | Base _|Pointer _ -> 1
```

Comment:
That one was properly written and the warning draw you attention onto the problematic situation to correct...